### ISSUE TITLE
Metadata QA dashboard feature request

### ISSUE THREAD

**Issue by catlu** [sciencehistory/chf-sufia#1101]
Putting in a departing feature request in as @HKativa suggested!

After login, there's a dashboard called "User Activity" that doesn't show any activity. (I remember when we were on Sufia 6, it showed the recent activities of various users, but it went away in Sufia 7.)

Currently, my workflow in QAing metadata from curators is that:

1) I go into this page--https://digital.sciencehistory.org/users, and every once in a while I click on the usernames of various curators to see if that have any recent activity for me to check.

and/or

2) I change the search results page to a recently modified sort and see what the recently updated collections/objects are.

It's a very unorganized process, and it'd be nice to have a functioning "User Activity" dashboard back, so I don't have to go to individual profiles to check on a random basis. And then ideally, it would be awesome if the dashboard also showed what the changes were. Currently, the information in User Activity looks like: 

> User clu@sciencehistory.org has updated [Bois de Boulogne](https://digital.sciencehistory.org/works/pz50gw93x) | 1 minute ago

It would be great if it could at least tell me which fields were updated, or even something like:

> Identifier, Accession No. was changed from "2016.10" to "2016.010".

**Comment by jrochkind** [comment]
I think it's unlikely we can do this in the current stack, but it's super useful to have it recorded here nonetheless, as a target, for future possibilities.  Thanks @catlu!

**Comment by jrochkind** [comment]
Would it be sufficient if this was just a universal log that looked the same to everyone, or would it be important for you to be able to mark certain activities as 'seen', like a personal inbox? If that makes sense.

**Comment by jrochkind** [comment]
I think what you want is something I wouldn't really call "user activity", what you want is **object/metadata changes** activity. Maybe?

**Comment by eddierubeiz** [comment]
Just chatted with @catlu  in person, and it appears that a very simple list showing 1) which records were recently changed, and b) who made those changes would replace a great deal of the lost functionality.

**Comment by catlu** [comment]
You're right about the "object/metadata changes activity @jrochkind, I was more stuck on the old Sufia terminology/concept. I think a universal log would be pretty sufficient, actually! Personal inbox doesn't seem necessary.

**Comment by jrochkind** [comment]
Excellent, because universal log is a lot easier. Although still I think not neccesarily in _this_ stack. But I have ideas for my future fantasy stack, that I have not given up hope on getting to.
